spec/intro.dd

```diff
@@ -10,59 +10,58 @@ to create native libraries or executables.)
 $(P This document is the reference manual for the D Programming Language. For more information and
 other documents$(COMMA) see $(LINK2 https://dlang.org/, The D Language Website).)
 
-$(H2 Phases of Compilation)
+$(H2 Conceptual models)
 
-$(P The process of compiling is divided into multiple phases. Each phase has no
-dependence on subsequent phases. For example, the scanner is not perturbed by
-the semantic analyzer. This separation of the passes makes language tools like
-syntax directed editors relatively easy to produce. It also is possible to
-compress D source by storing it in $(SINGLEQUOTE tokenized) form.)
+$(H3 Environments)
 
-$(OL
-        $(LI $(B source character set)$(BR)
+$(P The data-processing-system in which a D program is translated from source to executable format
+will be called the translation environment and the data-processing-system in which the translated
+D program is executed will be called the execution environment. The translation environment and
+the execution environment need not have the same system architecture.)
```

Comment:
I think this paragraph is playing with words a bit too much and it seems the only purpose is to have a heading called "environments"? The first data-processing-system *is* the compiler, so just call it that and hoist this paragraph in to two sections: Compiler, and Execution environment (also makes it much more obvious IMO). And then if you must, throw in linker in the compiler to denote the part that create the exe, but maybe that's too much detail for now.

Reply:
Simplified.